src/telemetry.ts

```diff
@@ -46,13 +46,25 @@ export function createTrackingEvent(name: string, properties = {}): TelemetryEve
   }
 }
 
+function createTelemetryProperties(commandId: string, message?: string): TelemetryProperties {
+  const telemetryProps: TelemetryProperties = telemetryProperties(commandId);
+  if (message) telemetryProps['message'] = hideClusterInfo(message);
+  return telemetryProps;
+}
+
+export function telemetryClusterInfo(commandId: string, message?: string): void {
```

Comment:
I was thinking to have common function for sending telemetry data. 
I don't think that having separate function for every possible telemetry data is good.

Also it is not clear what `commandId` means in context of this function.